Detectors/MUON/MID/Workflow/src/RawDecoderSpec.cxx

```diff
@@ -88,13 +89,19 @@ class RawDecoderDeviceDPL
 
 of::DataProcessorSpec getRawDecoderSpec(bool isDebugMode)
 {
-  return getRawDecoderSpec(isDebugMode, FEEIdConfig(), CrateMasks(), ElectronicsDelay(), 0);
+  return getRawDecoderSpec(isDebugMode, FEEIdConfig(), CrateMasks(), ElectronicsDelay());
 }
 
-of::DataProcessorSpec getRawDecoderSpec(bool isDebugMode, const FEEIdConfig& feeIdConfig, const CrateMasks& crateMasks, const ElectronicsDelay& electronicsDelay, size_t subSpec)
+of::DataProcessorSpec getRawDecoderSpec(bool isDebugMode, const FEEIdConfig& feeIdConfig, const CrateMasks& crateMasks, const ElectronicsDelay& electronicsDelay, long int subSpec)
 {
-  header::DataHeader::SubSpecificationType subSpecType(subSpec);
-  std::vector<of::InputSpec> inputSpecs{of::InputSpec{"mid_raw", of::ConcreteDataTypeMatcher{header::gDataOriginMID, header::gDataDescriptionRawData}, of::Lifetime::Timeframe}};
+  header::DataHeader::SubSpecificationType subSpecType(subSpec < 0 ? 0 : subSpec);
```

Comment:
I'd say that there's a potential type size mismatch here: SubSpecificationType is uint32_t while long (generally) is 8 bytes-long (except on e.g. Microsoft C++ where it is indeed 4 bytes, it seems)

Reply:
Should be fixed in the new commit (to be squashed)

Reply:
@aphecetche : strange: I added a commit in the branch but it is not popping up here...is this normal?